playbooks/streisand.yml

```diff
@@ -20,7 +20,8 @@
       when: streisand_openvpn_enabled
     - role: shadowsocks
       when: streisand_shadowsocks_enabled
-    - ssh
+    - role: ssh-forward
+      when: streisand_ssh_forward_enabled
     - role: tinyproxy
       when: streisand_tinyproxy_enabled
```

Comment:
Since tinyproxy is dependent on the SSH forwarding user:

```
role: tinyproxy
when: streisand_tinyproxy_enabled and streisand_ssh_forward_enabled
```